shared/clarifai_shared/config.py

```diff
@@ -63,15 +117,19 @@ class ClarifAIConfig:
     )
     neo4j: DatabaseConfig = field(default_factory=lambda: DatabaseConfig("", 0, "", ""))
 
+    # New configuration sections
+    embedding: EmbeddingConfig = field(default_factory=EmbeddingConfig)
+    concepts: ConceptsConfig = field(default_factory=ConceptsConfig)
+
     # Message broker configuration
     rabbitmq_host: str = "rabbitmq"
     rabbitmq_port: int = 5672
     rabbitmq_user: str = "user"
     rabbitmq_password: str = ""
 
     # Service configuration
-    vault_path: str = "/vault"
-    settings_path: str = "/settings"
+    vault_path: str = "/vault"  # Deprecated: use paths.vault
+    settings_path: str = "/settings"  # Deprecated: use paths.settings
```

Comment:
This is all before an initial release, no need to deprecate. DO ensure this doesnt break any usage elsewhere in the project though.